tests/testthat/test-class2tree.R

```diff
@@ -11,6 +11,32 @@ spnames <- c("Klattia flava", "Trollius sibiricus", "Arachis paraguariensis",
 dupnames <- c("Mus musculus", "Escherichia coli",
               "Haloferax denitrificans", "Mus musculus")
 
+test_that("internal functions of class2tree", {
+  skip_on_cran() # uses secrets
+  vcr::use_cassette("class2tree_classification_call", {
+    out <- classification(spnames, db = "ncbi", messages = FALSE)
+  })
+  
+  rankList <- taxize:::dt2df(lapply(out, taxize:::get_rank), idcol = FALSE)
```

Comment:
you don't need to use `:::` for internal fxn calls, just call the fxn itself, so `get_rank` instead of `taxize:::get_rank`

Reply:
I have removed them. I don't know that within the test one don't have to use `:::` for not exported function.